src/commands/utility/info/emote.js

```diff
@@ -0,0 +1,35 @@
+const { Command, SwitchbladeEmbed, CommandError } = require('../../../')
+const moment = require('moment')
+
+module.exports = class InfoEmoji extends Command {
+  constructor (client) {
+    super({
+      name: 'emoji',
+      parent: 'info',
+      requirements: { guildOnly: true, botPermissions: ['MANAGE_EMOJIS'] },
+      parameters: [{
+        type: 'emoji',
+        missingError: 'commands:info.subcommands.emote.missingEmoji'
+      }]
+    }, client)
+  }
+
+  async run ({ t, author, channel, language }, emoji) {
+    moment.locale(language)
+    try {
+      const emojiAuthor = await emoji.fetchAuthor()
+      const embed = new SwitchbladeEmbed(author)
+        .setTitle(`${emoji.animated ? `${t('commands:info.subcommands.emoji.animatedTag')} ` : ''}${emoji.name}`)
+        .setDescription(`
+        **${t('commands:guildinfo.createdAt')}**: ${moment(emoji.createdAt).format('LLL')}\n(${moment(emoji.createdAt).fromNow()})
+        **ID**: \`${emoji.id}\` \`\`<:${emoji.name}:${emoji.id}>\`\`
+        ${emojiAuthor ? `**${t('commands:info.subcommands.emoji.whoAdded')}**: ${emojiAuthor.tag}` : ''}
+        `)
+        .setImage(emoji.url)
+
+      channel.send(embed)
+    } catch (e) {
+      throw new CommandError(t('errors:needManageEmojis'))
```

Comment:
Why is this needed?

Reply:
> I need help with the emote command, calling `emoji.fetchAuthor()` is making a missing error while I do put a botPermission of `MANAGE_EMOJIS`

> I'll add the function inside a try catch since there's no other way.

Reply:
What I believe is happening is that the bot is ignoring command permissions, I'll look into it a bit more

Reply:
I've understood the error. It's trying to verify the permissions on the channel that the message is being sent, rather than verifying it on the guild itself. I'll try to make a fix locally, and I'll push to this branch

Reply:
Oh, I see. It's trying to verify if the bot has permissions, but opts.permissions doesn't exists, that's why it's returning true every time. Blame @davipatury.
![image](https://user-images.githubusercontent.com/32089154/92314899-36398300-efd6-11ea-8843-be45610bf588.png)

Reply:
Yep, now working as intended. 
![image](https://user-images.githubusercontent.com/32089154/92314905-481b2600-efd6-11ea-9bb9-59710411ccf8.png)